pkg/internal/test/starts_with_matcher.go

```diff
@@ -0,0 +1,54 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/types"
+)
+
+// StartWith is a GomegaMatcher verifying if given string starts with prefix
+func StartWith(expected string) types.GomegaMatcher {
+	return &startsWith{expected}
+}
+
+type startsWith struct {
+	prefix string
+}
+
+func (matcher *startsWith) Match(actual interface{}) (success bool, err error) {
+	actualString, ok := toString(actual)
+	if !ok {
+		return false, fmt.Errorf("StartsWith matcher requires a string or stringer.  Got:\n%s", format.Object(actual, 1))
+	}
+
+	return strings.Contains(actualString, matcher.prefix), nil
```

Comment:
shouldn't that be `strings.HasPrefix` instead of the `Contains` when the matcher is named `StartWith`?

Reply:
Good catch, not sure why it's not `HasPrefix`

Reply:
Fixed with 09dadcc